src/_gettsim/transfers/erwerbsm_rente.py

```diff
@@ -0,0 +1,228 @@
+from _gettsim.shared import dates_active
+
+
+@dates_active(start="2001-01-01")
+def erwerbsm_rente_m(
+    erwerbsm_rente_zugangsfaktor: float,
+    entgeltp_erwerbsm_rente: float,
+    rentenwert: float,
+    rentenartfaktor_erwerbsm_rente: float,
+    ges_rente_vorauss_erwerbsm: bool,
+) -> float:
+    """Calculating the Erwerbsminderungsrente (public disability insurance claim)
+
+    Legal reference: SGB VI § 64: Rentenformel für Monatsbetrag der Rente
+
+
+    Parameters
+    ----------
+    erwerbsm_rente_zugangsfaktor
+        See :func:`erwerbsm_rente_zugangsfaktor`.
+    entgeltp_erwerbsm_rente
+        See :func:`entgeltp_erwerbsm_rente`.
+    rentenwert
+        See :func:`rentenwert`.
+    rentenartfaktor_erwerbsm_rente
+        See :func:`rentenartfaktor_erwerbsm_rente`.
+    ges_rente_vorauss_erwerbsm
+        See :func:`ges_rente_vorauss_erwerbsm`.
+    Returns
+    -------
+    Erwerbsminderungsrente (public disability insurance claim)
+
+    """
+
+    if ges_rente_vorauss_erwerbsm:
+        out = (
+            entgeltp_erwerbsm_rente
+            * erwerbsm_rente_zugangsfaktor
+            * rentenwert
+            * rentenartfaktor_erwerbsm_rente
+        )
+    else:
+        out = 0.0
+    return out
+
+
+def ges_rente_vorauss_erwerbsm(
```

Comment:
```suggestion
def ges_rente_anspruch_erwerbsm(
```

? I would find that clearer and you are using it inside the function, too. But once we can write it out as `voraussetzung` I am fine with it (won't be long).

Reply:
ges_rente_vorauss_vorzeitig
ges_rente_vorauss_regelrente
ges_rente_vorauss_frauen
ges_rente_vorauss_langj
...
are the names of functions checking the eligibility for other pension types. I wanted to match that pattern